FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Core/Tasks/TaskExecutor.cs

```diff
@@ -145,5 +116,50 @@ orderby dayGroups.Key
 
             return TaskResult.CreateFromTaskContext(taskContext);
         }
+
+        private async Task ExecuteInternalAsync(
+            JsonBatchData inputData,
+            TaskContext taskContext,
+            DateTime dateTime,
+            string continuationToken,
+            CancellationToken cancellationToken = default)
+        {
+            foreach (var schemaType in taskContext.SchemaTypes)
+            {
+                var processParameters = new ProcessParameters(schemaType);
+
+                var parquetStream = await _parquetDataProcessor.ProcessAsync(inputData, processParameters, cancellationToken);
+                var skippedCount = inputData.Values.Count() - parquetStream.Count;
+
+                if (parquetStream?.Value?.Length > 0)
+                {
+                    // Upload to blob and log result
+                    var blobUrl = await _dataWriter.WriteAsync(parquetStream, taskContext, dateTime, cancellationToken);
+                    taskContext.PartId[schemaType] += 1;
+
+                    var batchResult = new BatchDataResult(
+                        taskContext.ResourceType,
+                        continuationToken,
+                        blobUrl,
+                        inputData.Values.Count(),
+                        inputData.Values.Count() - skippedCount);
+                    _logger.LogInformation(
```

Comment:
Shall we add schema information here for logging?

Reply:
Added schemaType in formation in **BatchDataResult**